sentinelsat/sentinel.py

```diff
@@ -289,6 +290,52 @@ def get_footprints(self, products):
             )
         return geojson.FeatureCollection(feature_list)
 
+    def to_dict(self, products):
+        """Return the products in a dictionary object with the values in their appropriate Python types"""
+        try:
+            from osgeo import ogr
+        except ImportError:
+            raise ImportError('Python OGR library does not found')
+        output = {}
+        for prod in products:
+            prodname = prod['title']
+            output[prodname] = {}
+            for key in prod.keys():
+                if key != 'title' and type(prod[key]) in [types.UnicodeType, types.StringType]:
```

Comment:
This does not work in Python 3, only Python 2. Everything we do should support Python 3.